### ISSUE TITLE
plotdist: passing graphical parameters

### ISSUE THREAD

**Issue by ucb** [aursiber/fitdistrplus#27]
I would like to reduce the x-axis range for the histogram part of the plot

````
plotdist(tmp[,(Sim.Val)],distr = "lnorm",para=list(meanlog=tmp[,mean(log(Sim.Val))], sdlog=tmp[,sd(log(Sim.Val))]),
  demp = TRUE,xlim=c(0,200))
````

However, specifyling the `xlim` kills two out of four graphs and trims both hist and Q-Q. How does not pass differentiated graphical parameters?

Furthermore, passing `breaks` as a vector (something I am used to doing when using `hist` directly) throws an error as in some function the check is implemented as 
`if (breaks == "default") h <- hist(data, plot = FALSE) else h <- hist(data,...`
and not
`if (identical(breaks,"default")) h <- hist(data, plot = FALSE) else h <- hist(data,...`

**Comment by aursiber** [comment]
Thanks for your report.
The `breaks` bug in `plotdist` was solved in 9fbdd72e85c55e9b37bd7e7900f6bdf931aaac3e.

About differential graphical parameters, we'll look into your suggestion soon.

**Comment by aursiber** [comment]
The `plotdist` function should be used for exploratory purposes. You can customize each of the four sub-graphics separately as described in [section 5.2 of the FAQ](https://lbbe-software.github.io/fitdistrplus/articles/FAQ.html#how-to-personalize-goodness-of-fit-plots).
